src/components/RecoveryPassword/RecoveryPassword.jsx

```diff
@@ -0,0 +1,78 @@
+const RecoveryPassword = () => {
+  return (
+    <div className="card ">
+      <h5 className="card-header p-3 d-flex justify-content-center  ">
+        ¿Olvidaste tu contraseña?
+      </h5>
+      <div className="card-body py-5 ">
+        <div className="row justify-content-md-center">
+          <div className="">
+            <form action="" className="">
```

Comment:
```suggestion
         <div className="">
            <form action="" className="">
```

Los atributos vacios deben ser eliminados